dataeng/jobs/analytics/SnowflakeReplicaImportFromS3.groovy

```diff
@@ -1,37 +1,91 @@
 package analytics
 import static org.edx.jenkins.dsl.AnalyticsConstants.common_multiscm
 import static org.edx.jenkins.dsl.AnalyticsConstants.common_parameters
-import static org.edx.jenkins.dsl.AnalyticsConstants.to_date_interval_parameter
 import static org.edx.jenkins.dsl.AnalyticsConstants.common_log_rotator
 import static org.edx.jenkins.dsl.AnalyticsConstants.common_wrappers
 import static org.edx.jenkins.dsl.AnalyticsConstants.common_publishers
-import static org.edx.jenkins.dsl.AnalyticsConstants.common_triggers
 
 class SnowflakeReplicaImportFromS3 {
     public static def job = { dslFactory, allVars ->
-        allVars.get('SNOWFLAKE_READ_REPLICA_IMPORTS').each { db, db_config ->
-            dslFactory.job("snowflake-${db.toLowerCase()}-read-replica-import-from-s3") {
+
+        Map jobConfigs = [
+            CREDENTIALS: [
+                CLUSTER_NAME: 'ImportCredentialsReadReplicaSnowflakeFromS3',
+                SCHEMA: 'CREDENTIALS_RAW',
+                SCRATCH_SCHEMA: 'CREDENTIALS_SCRATCH_LOADING',
+                DATABASE: 'credentials',
+                INCLUDE: '--include ["core_user", "django_session", "social_auth_usersocialauth"]',
+            ],
+            LMS: [
+                CLUSTER_NAME: 'ImportLMSReadReplicaSnowflakeFromS3',
+                SCHEMA: 'LMS_RAW',
+                SCRATCH_SCHEMA: 'LMS_SCRATCH_LOADING',
+                DATABASE: 'wwc',
+                INCLUDE: '--include ["assessment_assessmentd", "assessment_assessmentfeedbackd", "assessment_assessmentpartd", "auth_userprofiled", "certificates_generatedcertificated", "course_goals_coursegoald", "course_overviews_courseoverviewtabd", "edxval_videoimaged", "enterprise_enterprisecustomerd", "enterprise_historicalenterprisecustomerd", "entitlements_courseentitlementd", "problem_builder_answerd", "proctoring_proctoredexamstudentattemptd", "social_auth_usersocialauthd", "student_courseenrollmentd", "student_loginfailuresd", "student_sociallinkd", "submissions_scoresummaryd", "user_api_usercoursetagd", "wiki_articled", "wiki_articlerevisiond", "workflow_assessmentworkflowd"]',
```

Comment:
Curious if there is a reason why these table names have a "d" at the end.

Reply:
hah!  must have been a typo in my macro.